apps/mobile-wallet/src/features/localization/relativeTime.ts

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+type RelativeTimeEntries = {
+  future: string
+  past: string
+  s: string
+  m: string
+  mm: string
+  h: string
+  hh: string
+  d: string
+  dd: string
+  M: string
+  MM: string
+  y: string
+  yy: string
+}
+
+// See https://day.js.org/docs/en/customization/customization
+// and https://github.com/iamkun/dayjs/tree/dev/src/locale
+export const shortRelativeTime: Record<string, RelativeTimeEntries> = {
+  en: {
```

Comment:
No way to add those in Crowdin, through a dedicated json file?

Reply:
I decided to completely remove it, dayjs provides sane defaults for relative time, no need to add extra config on our side: https://github.com/iamkun/dayjs/tree/dev/src/locale